src/glyphs.js

```diff
@@ -111,6 +135,61 @@ const updateMessages = (id, path, req) => {
   )
 };
 
+const importMessages = (line) => {
+  const message = line.split(",");
+  const viewMessage = (message[1] === "Intro");
+
+  let path = "";
+  let newMessage = {"text": message[2]};
+
+  if (viewMessage) newMessage.probability = Number(message[3]);
+  else newMessage.probability = message.splice(3, 5).map(num => Number(num));
+
+  if (viewMessage) path = "view";
+  else path = `view.gauges.${message[1]}`;
+
+  return db.collection.updateOne(
+    {
+      "view.name": message[0].toLowerCase()
+    },
+    {
+      $addToSet: {
+        [`${path}.messages`]: newMessage
+      }
+    }
+  )
+}
+
+const overwriteMessages = (file) => {
+  let first = true;
+  let overwritten = [];
+
+  lineReader.eachLine(file, function(line) {
+    if (first) first = false;
+    else {
+      const message = line.split(",");
+      let path = "";
+
+      if (message[1] === "Intro") path = "view";
+      else path = `view.gauges.${message[1]}`;
+
+      if (!overwritten.includes(message[0] + path)) {
+        db.collection.updateOne(
+          {
+            "view.name": message[0].toLowerCase()
+          },
+          {
+            $set: {
+              [`${path}.messages`]: []
+            }
+          }
+        )
+        overwritten.push(message[0] + path);
```

Comment:
Should probably make `message[0]` case insensitive here, no?
Also why do we not allow overwriting the same message multiple times?  And even if we have this behavior, shouldn't we report to the user if they have a somewhat invalid request?